backend/controllers/plans.js

```diff
@@ -0,0 +1,67 @@
+
+
+const Plan = require('../models/Plan')
+const Task = require('../models/Task')
+const User = require('../models/User')
+const mongoose = require('mongoose');
+const ObjectId = mongoose.Types.ObjectId;
+
+const {retErr} = require('../utils/index');
+const {checkInputs} = require('../utils/index')
+
+
+exports.findPlanToOwner = async (req, res) => {
```

Comment:
Ich würde es `findByOwner` nennen. Oder noch besser  `show`, wie es in REST üblich ist. https://blog.usejournal.com/rails-and-the-seven-restful-routes-838e3e5e85d9